api-test.py

```diff
@@ -0,0 +1,144 @@
+import requests
+from PIL import Image
+import io
+import base64
+import time
+from PIL import Image
+from datetime import datetime
+import sys
+from constants import ModelName
+
+# endpoint = "http://nicheimage.nichetensor.com/api/v1"
+endpoint = "http://localhost:8000/api/v1"
+
+headers = {
+    "API_KEY": "poih34p89bewrfgf3",
```

Comment:
is it just a dummy api-key?

Reply:
It's just a dummy api-key. We can use it as an environment variable or we can get the api-key from user input.

FYI, this api-test.py is a test file to test the API functions on the local environment.
We can add api-test.py to the gitignore list.